src/trait-bounds.md

```diff
@@ -77,8 +77,8 @@ Trait and lifetime bounds are also used to name [trait objects].
 
 ## `?Sized`
 
-`?` is only used to declare that the [`Sized`] trait may not be
-implemented for a type parameter or associated type. `?Sized` may
+`?` is only used to declare that the [`Sized`] trait may be
+unimplemented for a type parameter or associated type. `?Sized` may
```

Comment:
I think it might be better to say "is not required" instead of "may be unimplemented". Maybe add a parenthetical "(Type parameters and associated types have an implicit `Sized` bound.)" after this sentence.